### PR TITLE
mint by final dcx amount

### PR DESCRIPTION
**Pull request by LorranSutter**

### CODE REVIEW

File: src/interface/IDimoCredit.sol [hunk 1]
```diff
@@ -7,6 +7,7 @@ interface IDimoCredit {
     function burn(address from, uint256 amount) external;
     function transferFrom(address from, address to, uint256 value) external returns (bool);
     function mintAmountDc(address to, uint256 dimoCredits, bytes calldata data) external returns (uint256 amountIn);
-    function mint(address to, uint256 amountIn, bytes calldata data) external returns (uint256 dimoCredits);
+    function mintInDimo(address to, uint256 amountDimo, bytes calldata data) external returns (uint256 dimoCredits);
+    function mint(address to, uint256 amountDimoCredits, bytes calldata data) external;
```
Comment (elffjs) [inline src/interface/IDimoCredit.sol:22]: This seems like a breaking change for the `mint` function, doesn't it? I feel like we can't do this.

File: src/interface/IDimoCredit.sol [hunk 2]
```diff
@@ -7,6 +7,7 @@ interface IDimoCredit {
     function burn(address from, uint256 amount) external;
     function transferFrom(address from, address to, uint256 value) external returns (bool);
     function mintAmountDc(address to, uint256 dimoCredits, bytes calldata data) external returns (uint256 amountIn);
```
Comment (elffjs) [inline src/interface/IDimoCredit.sol]: Do we even have this function?

File: src/interface/IDimoCredit.sol (hunk 1 above)
Comment (elffjs) [reply src/interface/IDimoCredit.sol:22]: I guess the question is whether anyone is calling this. Who is there? Dev console? Can you check what they're using?

Comment (LorranSutter) [reply src/interface/IDimoCredit.sol:22]: Oh yeah I am aware of that. Pretty much dev console so far. I would have them update it before deploying.
To clarify, I think keeping `mint` for DCX amout makes more sense and make the distinction for DIMO tokens.

File: src/interface/IDimoCredit.sol (hunk 2 above)
Comment (LorranSutter) [reply src/interface/IDimoCredit.sol]: Not really. Let me remove that.

**Pull request merged**